src/app/reducers/vm/redux/vm.reducers.ts

```diff
@@ -43,6 +43,7 @@ export interface State extends EntityState<VirtualMachine> {
 
 export interface FormState {
   loading: boolean;
+  loaded: boolean;
```

Comment:
Do we really need this for a form? This looks like a derivative data from loading it this case.

Reply:
If it's not loading it may be that it hasn't started loading yet.